### PR TITLE
ru-RU translation

### PR DESCRIPTION
**Pull request by ra30r**
Pomax, 

Thank you for explicit instructions. I have played around with repos following your guides and accomplished moving 3 latest commits into separate branch; hard resetting / force pushing to your official master into my origin master and merging the above 2 to resolve conflicts. It has deleted original pull request at the point of force push. I'll pay attention to pulling from origin before making new pull request onto it. And I could also do a pull request from separate branch if thats anyhow beneficial or implied. Lastly — requested file changes (e.g. title name etc.) where incorporated before proceeding with all of this. Let me know of anything else that needs correction. Planning on translating more chapters later on this week.

### CODE REVIEW

**Comment by Pomax** [comment]
Let me just say I massively appreciate you sticking with it to get these translations in. I remember how scary my first rebases were, and the fact that you've powered through them puts you above most folks!

File: docs/chapters/explanation/content.ru-RU.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  x = \cos(t) \\
+  y = \sin(t)
+\end{matrix} \right.
+\]
+
+Так-то. Координаты <i>x</i>/<i>y</i>. Связанные мистическим значением <i>t</i>.
+
+Таким образом, мы видим, что параметрические функции не определяют значение <i>y</i> через значение <i>x</i>, как обычные ф-ции, вместо этого они выводят и <i>x</i> и <i>y</i> из мистического значения <i>t</i>. Значит, для каждого значения <i>t</i>, существуют два соответствующих значения <i>x</i> и <i>y</i>, которые мы можем использовать для зарисовки кривой на графике. К примеру, приведенная выше параметрическая функция выводит координаты точки на круге: можно подставить любое значение <i>t</i> (от негативной до позитивной бесконечности), и на выводе мы всегда получим координаты точки на круге с центром в (0,0) и радиусом в 1. Зарисовав точки для <i>t</i> от 0 до 5, получаем следующее:
+
+<graphics-element title="A (partial) circle: x=sin(t), y=cos(t)" src="./circle.js">
```
Comment (Pomax) [inline docs/chapters/explanation/content.ru-RU.md:44]: this probably still needs a russian title

File: docs/chapters/weightcontrol/content.ru-RU.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Это создает неожиданный эффект: превращает наш полиноминал во что-то, что полиноминалом более не является. Теперь этот тип кривой есть супер классом полиноминала, что позволяет производить крутые вещи, на которые кривые Безье не способны сами по себе. Например обозначить круг посредством такой кривой (что, как мы убедимся чуть погодя, категорически невозможно используя стандартную кривую Безье).
+
+Но лучшим объяснением, как и всегда, является наглядная демонстрация: давайте посмотрим на эффект "соотносительности" нашей кривой Безье на примере интерактивного графика соотнесительной кривой. Следующий график показывает кривую Безье из предыдущей секции, "обогащенную" факторами соотношений для каждой координаты. Чем ближе к нулю мы выставляем значения этих факторов — тем меньше влияния оказывают соответственные контрольные точки на начертания кривой, и, соответственно, чем выше эти показатели — тем больше влияния контрольная точка оказывает на изгиб кривой. Попробуйте сами:
+
+<graphics-element title="Our rational cubic Bézier curve" src="./rational.js">
```
Comment (Pomax) [inline docs/chapters/weightcontrol/content.ru-RU.md:25]: `title` attribute

**Review by Pomax — CHANGES_REQUESTED** [review]
two titles still need translating, but after that I think we can merge this in!

**Comment by ra30r** [comment]
fixed titles, added another chapter

**Review by Pomax — APPROVED** [review]

**Comment by Pomax** [comment]
nice, in it goes!

**Comment by Pomax** [comment]
It looks like there were a few "double newlines" in the markdown, which throws off the conversion. To help us catch that in the future, I've added CI testing for pull requests, and I've filed https://github.com/Pomax/BezierInfo-2/issues/296 as a task to add automatic markdown reformatting during the build step (similar to `prettier` for the js)

**Comment by ra30r** [comment]
> It looks like there were a few "double newlines" in the markdown, which throws off the conversion. To help us catch that in the future, I've added CI testing for pull requests, and I've filed #296 as a task to add automatic markdown reformatting during the build step (similar to `prettier` for the js)

Sorry about that, didn't catch it since using regular text editor for spell check highlights. CI testing is nice, though. Sort of fancy too — installing tex-live on the go..  the only version I found for mac so far was like 6 gb.

**Comment by Pomax** [comment]
I think you can use `brew` to only install the parts of TeX Live that you need, too, but I just did a full texlive install on my own mac (I mostly work on Windows, but I have a mac for additional testing) as I stuck a fairly large SSD in it so space is not an issue.

**Comment by Pomax** [comment]
ah, no. It looks like for mac, the preferred solution is [mactex](https://www.tug.org/mactex/)

**Pull request merged**